tests/test_backbone.py

```diff
@@ -0,0 +1,132 @@
+import pytest
+import torch
+
+from mmdet.models.backbones import ResNet, ResNetV1d, ResNeXt
+
+
+def test_basic_block():
+    inputs = torch.rand(2, 3, 224, 224)
+    self = ResNet(depth=18)
+    level_outputs = self.forward(inputs)
+    for level_out in level_outputs:
+        print(tuple(level_out.shape))
```

Comment:
Check the shape with expected ones instead of just printing it.

Reply:
Besides the output shapes, we also need to check the architecture of the constructed model.